### PR TITLE
OCPQE-22599: Create QE dedicated GCP cluster profile 'gcp-autorelease-qe'

### PR DESCRIPTION
**Pull request by jianli-wei**
ci-tools [openshift/ci-tools/pull/4152](https://github.com/openshift/ci-tools/pull/4152)

Profile | Profile String	| Cloud Type
-- | -- | --
ClusterProfileGCPAutoReleaseQE | gcp-autorelease-qe | gcp

Following credentials will be included:
Item | Source | Comments
-- | -- | --
pull-secret | TPDP platform | pull-secret
gce.json | Credentials in QE's Vault | GCP credentials of project "openshift-qe"
xpn_creds.json | Credentials in QE's Vault | GCP credentials of project "openshift-qe-shared-vpc" for GCP XPN testing
ssh-publickey | Credentials in QE's Vault | QE public ssh key
ssh-privatekey | Credentials in QE's Vault | QE private ssh key
other certs and credentials for servers | Credentials in QE's Vault |

### CODE REVIEW

File: core-services/ci-secret-bootstrap/_config.yaml [hunk 1]
```diff
@@ -1233,6 +1233,33 @@ secret_configs:
...
+      - auth_field: auth
+        email_field: email
+        item: registry.connect.redhat.com-pull-secret
+        registry_url: registry.connect.redhat.com
+      - auth_field: auth
+        email_field: email
+        item: registry.redhat.io-pull-secret
+        registry_url: registry.redhat.io
+  to:
+  - cluster_groups:
```
Comment (jianlinliu) [inline core-services/ci-secret-bootstrap/_config.yaml:1259]: I do not think we need the above `from` section, we just need to append the new cluster profile to 1235 line as a new array element.

File: core-services/prow/02_config/generate-boskos.py [hunk 2]
```diff
@@ -175,6 +175,9 @@
     'gcp-qe-quota-slice': {
         'us-central1': 30,
     },
+    'gcp-autorelease-qe-quota-slice': {
+        'us-central1': 4,
```
Comment (jianlinliu) [inline core-services/prow/02_config/generate-boskos.py:179]: Do we have other region which have enough quota to run ci testing ?

Comment (jianli-wei) [reply core-services/prow/02_config/generate-boskos.py:179]: It looks no for now. Shall I set up one?

Comment (jianlinliu) [reply core-services/prow/02_config/generate-boskos.py:179]: okay, if not yet, let us use the the existing one.

**Comment by jianlinliu** [comment]
@droslean Do you mind to review this PR?

**Comment by jianlinliu** [comment]
cc @psalajova to review and approve, thanks.

**Pull request merged**